src/runtime/client-config-manager.js

```diff
@@ -63,10 +68,28 @@ export class ClientConfigManager {
       this.getClientConfig();
     }
     return this.responsePromise_.then(
-      (clientConfig) => clientConfig.autoPromptConfig
+      (clientConfig) => clientConfig.autoPromptConfig || undefined
```

Comment:
Could you update the `getAutoPromptConfig` method's return type, to mention it can return a `Promise` containing `undefined`?

Reply:
Done.